proto/pulumi/resource.proto

```diff
@@ -36,6 +36,7 @@ service ResourceMonitor {
     rpc RegisterResourceOutputs(RegisterResourceOutputsRequest) returns (google.protobuf.Empty) {}
 
     rpc RegisterStackTransform(Callback) returns (google.protobuf.Empty) {}
+    rpc RegisterInvokeTransform(Callback) returns (google.protobuf.Empty) {}
```

Comment:
This should probably be RegisterStackInvokeTransform and we should document the method above as being a resource transform.